### DISCUSSION TITLE
Feature request: Toot to timeline of a particular instance

### DISCUSSION THREAD

**Discussion by mgloec** [mastodon/mastodon#22754] [General]
I want to post about a medical topic public. But I am a medical amateur... honestly lower than that. So I want to post additionally to medical professionals instance, to get response if I am totally wrong in my topic argumentation. I know that this idea is maybe more complex. Because posting to a entire instance should not effect the local timeline of the instance. But maybe this can be realized in a new timeline, to which members of the target instance can opt in?

**Comment by mjankowski** [comment]
I realize this is three years old, but -- the simplest way to do something like this is probably just use tags on your post? They'll wind up in the feeds of (federated) users following those tags.
